packages/orchestration/src/util/filtering.ts

```diff
@@ -33,13 +38,22 @@ export function buildAzureContentFilter(
  * @returns Filter config object.
  */
 export function buildAzureContentSafetyFilter(
```

Comment:
[req] The func name in deprecation message is wrong:  @deprecated Since 1.8.0. Use {@link buildAzureContentSafety()} instead.